lib/msal-angular/src/msal.service.ts

```diff
@@ -29,9 +32,10 @@ export class MsalService implements IMsalService {
         if (hash) {
             this.redirectHash = `#${hash}`;
         }
+        this.logger = this.instance.getLogger();
```

Comment:
I think instead of getting the existing msal-browser logger we should create a new logger with the same options, that way the log messages coming from `msal-angular` and coming from `msal-browser` are properly distinguished with relevant versions, etc.

Reply:
Done!